tar/tarsnap.1-mdoc.in

```diff
@@ -664,7 +664,16 @@ option specified in a configuration file.
 .Fl T ,
 or
 .Fl X )
-Filenames or patterns are separated by null characters,
+Synonym for
+.Fl -null-input .
+Kept for historical reasons and compatibility with other tar utilities.
```

Comment:
Not totally certain if you want to include this line in the man page, but I got the impression you'd want it, based on discussion elsewhere.